app/src/main/java/org/wikipedia/notifications/NotificationFilterItemView.kt

```diff
@@ -70,10 +71,16 @@ class NotificationFilterItemView constructor(context: Context, attrs: AttributeS
     }
 
     fun setSingleLabel(text: String) {
-        binding.notificationFilterLanguageCode.visibility = View.INVISIBLE
-        binding.notificationFilterWikiLogo.visibility = View.GONE
+        val textSize = 14f
+        binding.notificationFilterLanguageCode.visibility = View.GONE
+        binding.notificationFilterWikiLogo.visibility = View.VISIBLE
+        binding.notificationFilterWikiLogo.imageTintList =
+            ColorStateList.valueOf(ContextCompat.getColor(context, ResourceUtil.getThemedAttributeId(context, R.attr.colorAccent)))
+        binding.notificationFilterWikiLogo.setImageDrawable(AppCompatResources.getDrawable(context, R.drawable.ic_add_gray_themed_24dp))
         binding.notificationFilterCheck.visibility = View.GONE
-        binding.notificationFilterTitle.text = text
+        binding.notificationFilterTitle.setTextColor(ResourceUtil.getThemedColor(context, R.attr.colorAccent))
+        binding.notificationFilterTitle.text = text.uppercase()
```

Comment:
Looks like the `TypeFace` should be `Roboto-Medium`. Now it is in regular Roboto.